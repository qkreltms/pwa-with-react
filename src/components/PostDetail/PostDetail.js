import React from 'react'
import Post from '../Post/Post'

const PostDetail = (props) => {
  return (
    <div>
      <Post {...props.post} />
    </div>
  )
}

export default PostDetail
