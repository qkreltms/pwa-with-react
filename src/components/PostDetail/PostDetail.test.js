/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import PostDetail from './PostDetail'
import toJson from 'enzyme-to-json'

test('About component should render as expected', () => {
  const component = shallow(<PostDetail />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
