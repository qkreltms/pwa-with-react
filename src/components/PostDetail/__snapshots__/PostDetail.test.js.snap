// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render as expected 1`] = `
<div>
  <Post />
</div>
`;
